### PR TITLE
MTDSA-18330 - release 4 changelog

### PR DESCRIPTION
**Pull request by rhysconnor**
subject to change, will add breaking change in on release day!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,48 @@ You should now receive an email approximately every two weeks containing the cha
...
+  * `Retrieve a Self Assessment Tax Calculation`
+
+#### self-employment-business-api
+
+* New `DYNAMIC` Gov-Test-Scenarios added to v3.0 endpoint:
+  * `Retrieve a Self-Employment Period Summary`
+
+#### individuals-expenses-api
+
+* Code improvement for breaking change, The code changes are for V2:
```
Comment (ben-HMRC) [inline README.md:72]: "Code improvement for breaking change. The code changes are for v2:"

File: README.md [hunk 2]
```diff
@@ -39,6 +39,48 @@ You should now receive an email approximately every two weeks containing the cha
...
+  * A new `RULE_INVALID_SUBMISSION_PENSION_SCHEME` error has been added.
+
+
+#### mtd-sa-test-support-api
+
+* New feature `Create`, `Restore`, `List` and `Delete Checkpoint`. These endpoints allow a developer to create, restore, list and delete a checkpoint for the stateful data for a particular NINO.
```
Comment (ben-HMRC) [inline README.md:78]: "These endpoints allow a developer to" -> "These endpoints enable you to"

File: README.md [hunk 3]
```diff
@@ -39,6 +39,48 @@ You should now receive an email approximately every two weeks containing the cha
...
+* New feature `Create`,  `Delete Test Business`. These endpoints will be used by vendors to create & delete test businesses of their preferred choice.
```
Comment (ben-HMRC) [inline README.md:79]: "These endpoints will be used by vendors to create & delete test businesses of their preferred choice." -> "These endpoints enable you to create and delete test businesses as required."

File: README.md [hunk 4]
```diff
@@ -39,6 +39,48 @@ You should now receive an email approximately every two weeks containing the cha
...
+
+* Upgrade `HTTP Client` to Version 2
+* New `Auto delete checkpoint` feature, this feature will auto delete checkpoints after 7 days.
```
Comment (ben-HMRC) [inline README.md:82]: "feature, this feature" -> "feature. This feature"

**Review by LouiseWong98 — APPROVED** [review]

**Review by LouiseWong98 — APPROVED** [review]

**Review by LouiseWong98 — APPROVED** [review]

**Review by LouiseWong98 — APPROVED** [review]

**Pull request merged**